### ISSUE TITLE
IllegalStateException: WrappedPropertyDescriptor is not bound

### ISSUE THREAD

**Issue by ArtRoman** [Kotlin/kotlinx.serialization#768]
Can't migrate to kotlinx.serialization because of https://github.com/JetBrains/kotlin-native/issues/3900

Native (ios) build gives error on inherited abstract class with fields. There are just two simple classes to trigger an error in different files: 
Base.kt in commonMain:
```
import kotlinx.serialization.Serializable
@Serializable
abstract class Base {
    var id: Long = 0
}
```
and Extended.kt in commonMain:
```
import kotlinx.serialization.Serializable
@Serializable
class Extended : Base() {
    var details: String? = null
}
```

Stacktrace:

```> Task :app:linkDebugFrameworkIos FAILED
e: Compilation failed: org.jetbrains.kotlin.ir.descriptors.WrappedPropertyDescriptor@5ca50e74 is not bound

 * Source files: 
 * Compiler version info: Konan: 1.3.70 / Kotlin: 1.3.70
 * Output kind: FRAMEWORK

e: java.lang.IllegalStateException: org.jetbrains.kotlin.ir.descriptors.WrappedPropertyDescriptor@5ca50e74 is not bound
	at org.jetbrains.kotlin.ir.descriptors.WrappedDeclarationDescriptor.getOwner(WrappedDescriptors.kt:94)
	at org.jetbrains.kotlin.ir.descriptors.WrappedPropertyDescriptor.getContainingDeclaration(WrappedDescriptors.kt:827)
	at org.jetbrains.kotlin.resolve.DescriptorUtils.getContainingModuleOrNull(DescriptorUtils.java:215)
	at org.jetbrains.kotlin.resolve.DescriptorUtils.getContainingModule(DescriptorUtils.java:200)
	at org.jetbrains.kotlin.resolve.descriptorUtil.DescriptorUtilsKt.getModule(DescriptorUtils.kt:62)
	at org.jetbrains.kotlin.backend.konan.ir.IrProviderForInteropStubs.getDeclaration(IrProviderForInteropStubs.kt:23)
	at org.jetbrains.kotlin.backend.konan.ir.IrProviderForInteropStubs.getDeclaration(IrProviderForInteropStubs.kt:18)
	at org.jetbrains.kotlin.ir.util.ExternalDependenciesGeneratorKt.getDeclaration(ExternalDependenciesGenerator.kt:67)
	at org.jetbrains.kotlin.ir.util.ExternalDependenciesGenerator.generateUnboundSymbolsAsDependencies(ExternalDependenciesGenerator.kt:41)
	at org.jetbrains.kotlin.psi2ir.generators.ModuleGenerator.generateUnboundSymbolsAsDependencies(ModuleGenerator.kt:58)
	at org.jetbrains.kotlin.psi2ir.Psi2IrTranslator.generateModuleFragment(Psi2IrTranslator.kt:81)
…
```
It is very interesting that using both classes in single file doesn't cause error. So, this compiles as normal:
Merged.kt in commonMain
```
import kotlinx.serialization.Serializable
@Serializable
abstract class BaseSingle {
    var id: Long = 0
}

@Serializable
class ExtendedSingle : BaseSingle() {
    var details: String? = null
}
```

Marking the 'id' fiels in Base with @transient or removing it also makes the build process successful, but the field will be missing in json. You can make 'id' abstract and override it in extended class to make compilation successful. 

**Environment**
 - Kotlin version: 1.3.70
 - Library version: 0.20.0
 - Kotlin platforms: [JVM, Native (ios)]
 - Gradle version: 5.6.4, build tools 3.6.1
 - IDE version (if bug is related to the IDE) Android Studio 3.6.1
Build #AI-192.7142.36.36.6241897, built on February 27, 2020
Runtime version: 1.8.0_212-release-1586-b4-5784211 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
macOS 10.14.6

**Comment by RubenVot** [comment]
I came up with this exact exact same structure and exact same behaviour in my own project.

**Comment by sandwwraith** [comment]
Seems to be fixed in 1.7.x
